test/arithmetic/values/fixed_point_value_test.dart

```diff
@@ -0,0 +1,244 @@
+// Copyright (C) 2024 Intel Corporation
+// SPDX-License-Identifier: BSD-3-Clause
+//
+// 2024 September 24
+// Authors:
+//  Soner Yaldiz <soner.yaldiz@intel.com>
+
+import 'dart:math';
+import 'package:rohd/rohd.dart';
+import 'package:rohd_hcl/rohd_hcl.dart';
+import 'package:test/test.dart';
+
+void main() {
+  test('FXP: construct from sign,int,frac', () {
```

Comment:
nit: unnecessary to prefix test names with "FXP", since they're all in a fixed point test anyways.  If you do want to prefix, a `group` can do it.